primitives/runtime/src/traits.rs

```diff
@@ -32,11 +32,11 @@ use impl_trait_for_tuples::impl_for_tuples;
 use serde::{de::DeserializeOwned, Deserialize, Serialize};
 use sp_application_crypto::AppKey;
 pub use sp_arithmetic::traits::{
-	AtLeast32Bit, AtLeast32BitUnsigned, Bounded, CheckedAdd, CheckedDiv, CheckedMul, CheckedShl,
-	CheckedShr, CheckedSub, Ensure, EnsureAdd, EnsureAddAssign, EnsureDiv, EnsureDivAssign,
-	EnsureFixedPointNumber, EnsureFrom, EnsureInto, EnsureMul, EnsureMulAssign, EnsureOp,
-	EnsureOpAssign, EnsureSub, EnsureSubAssign, IntegerSquareRoot, One, SaturatedConversion,
-	Saturating, UniqueSaturatedFrom, UniqueSaturatedInto, Zero,
+	checked_pow, ensure_pow, AtLeast32Bit, AtLeast32BitUnsigned, Bounded, CheckedAdd, CheckedDiv,
```

Comment:
I've reexported `checked_pow`. Please, double-check if you agree with it.

Reply:
I think we will get rid of these anyway in https://github.com/paritytech/substrate/pull/12509

Reply:
I've read the issue, could be a problem with our last move of `ArithmeticError` to `sp_arithmetic`?

Reply:
No idea, I think its stale.